src/components/Card/Card.Styled.ts

```diff
@@ -35,23 +35,23 @@ export const Text = styled.div<{ isActive?: boolean }>`
 	right: 0;
 	bottom: 0;
 	padding: 6.5em 2em;
-	background-color: ${(props) => props.theme.colors.almostDark};
-	color: ${(props) => props.theme.colors.pureWhite};
+	background-color: ${({ theme }) => theme.colors.almostDark};
```

Comment:
almostDark, trochę to dziwnie brzmi. Jeżeli to prawie czarny, to może nazwać to dark + wratość liczbowa?

Czasami się tak robi, gray od 100 do 900 się ustala, co 100. gray100 to taki mega biały ledwo szary, a gray900 to prawie czarny. Wszystko co pomiędzy to stopniowe przejscie w inny kolor.

Konwencji na nazywanie kolorów jest sporo